examples/step-53/step-53.cc

```diff
@@ -46,6 +46,13 @@
 #include <boost/iostreams/filter/gzip.hpp>
 #include <boost/iostreams/device/file.hpp>
 
+// The last include file is required because we will be using a feature that is
+// not part of the C++11 standard. The deal.II library, however, can be installed
+// with a C++11 compliant compiler, without assuming that C++14 or C++17 features are
+// available. As some of these features are very useful, we provide their
```

Comment:
At least to my ears, this sounds a bit odd. 
Maybe just omit the sentence in the middle and start the last sentence with "As some of the C++14 features are very useful [...]"? In the end, there is also no need to mention C++17 here.